argcomplete/__init__.py

```diff
@@ -348,6 +343,30 @@ def collect_completions(self, active_parsers, parsed_args, cword_prefix, debug):
                             completions += subprocess.check_output(bashcomp_cmd).decode(sys_encoding).splitlines()
                         except subprocess.CalledProcessError:
                             pass
+
+            return completions
+
+        completions = []
+        """
+        for active_parser in active_parsers[:-1]:
+            debug('active_parser:', active_parser)
+            if self.always_complete_options or (len(cword_prefix) > 0 and cword_prefix[0] in active_parser.prefix_chars):
+                completions += get_option_completions(active_parser, cword_prefix)
+                debug('completions options:', completions)
+
+            completions = complete_active_option(active_parser, cword_prefix, parsed_args, completions)
+        """
```

Comment:
Why is this code block still here but commented?

I also worry that this seems like it might break completions in situations where more than one parser is in the active_parsers stack, and the last one is not the one that will consume the word under the cursor. I'll have to think more about when that's possible though (and write a test).